app/controllers/name_controller/create_and_edit_name.rb

```diff
@@ -253,19 +284,28 @@ def name_unlocked?
   end
 
   def minor_change?
+    return false if icn_id_changed?
+    return true if just_adding_author?
+
     old_name = @name.real_search_name
     new_name = @parse.real_search_name
     new_name.percent_match(old_name) > 0.9
   end
 
+  def icn_id_changed?
+    params[:name][:icn_id] != @name.icn_id
+  end
+
   def just_adding_author?
     @name.author.blank? && @parse.text_name == @name.text_name
   end
 
-  def email_admin_name_change
+  def email_admin_name_change(old_identifier: nil)
```

Comment:
It might be possible to avoid passing old_identifier around.  If email_admin_name_change is only ever called before the changes are saved, ActiveRecord provides access to the old value of icn_id.

Reply:
Thank you for the suggestion.
(I disliked adding the `old_identifier` flag: I don't like flags in the first place, and this method is already too complicated.)